internal/usecases/liquidity_provider/add_trusted_account.go

```diff
@@ -0,0 +1,39 @@
+package liquidity_provider
+
+import (
+	"context"
+
+	"github.com/rsksmart/liquidity-provider-server/internal/entities"
+	"github.com/rsksmart/liquidity-provider-server/internal/entities/liquidity_provider"
+	"github.com/rsksmart/liquidity-provider-server/internal/usecases"
+)
+
+type AddTrustedAccountUseCase struct {
+	trustedAccountRepository liquidity_provider.TrustedAccountRepository
+	signer                   entities.Signer
+	hashFunc                 entities.HashFunction
+}
+
+func NewAddTrustedAccountUseCase(
+	trustedAccountRepository liquidity_provider.TrustedAccountRepository,
+	signer entities.Signer,
+	hashFunc entities.HashFunction,
+) *AddTrustedAccountUseCase {
+	return &AddTrustedAccountUseCase{
+		trustedAccountRepository: trustedAccountRepository,
+		signer:                   signer,
+		hashFunc:                 hashFunc,
+	}
+}
+
+func (useCase *AddTrustedAccountUseCase) Run(ctx context.Context, account liquidity_provider.TrustedAccountDetails) error {
+	_, err := usecases.SignConfiguration(usecases.SetTrustedAccountId, useCase.signer, useCase.hashFunc, account)
```

Comment:
Why are you signing and then not using the result? We discussed this during the refinement of the epic, without the signature anyone can manipulate the trusted accounts directly from the DB. You need to store the signed version and check the integrity on every read.